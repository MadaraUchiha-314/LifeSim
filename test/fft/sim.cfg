
[bbv]
sampling = 0

[caching_protocol]
type = "parametric_dram_directory_msi"
variant = "mesi"

[clock_skew_minimization]
report = "false"
scheme = "barrier"

[clock_skew_minimization/barrier]
quantum = 100

[core]
spin_loop_detection = "false"

[core/cheetah]
enabled = "false"
max_size_bits_global = 36
max_size_bits_local = 30
min_size_bits = 10

[core/hook_periodic_ins]
ins_global = 1000000
ins_per_core = 10000

[core/light_cache]
num = 0

[dvfs]
transition_latency = 2000
type = "simple"

[dvfs/simple]
cores_per_socket = 1

[fault_injection]
injector = "none"
type = "none"

[general]
enable_icache_modeling = "true"
enable_pinplay = "false"
enable_signals = "false"
enable_smc_support = "false"
enable_syscall_emulation = "true"
inst_mode_end = "fast_forward"
inst_mode_init = "cache_only"
inst_mode_output = "true"
inst_mode_roi = "detailed"
issue_memops_at_functional = "false"
magic = "true"
num_host_cores = 0
output_dir = "/home/pr16/sniper/sniper-6.1/test/fft"
roi_script = "false"
suppress_stderr = "false"
suppress_stdout = "false"
syntax = "intel"
total_cores = 4

[hooks]
numscripts = 1
script0args = ""
script0name = "/home/pr16/sniper/sniper-6.1/test/fft/sim.scripts.py"

[instruction_tracer]
type = "none"

[log]
circular_log = "false"
disabled_modules = ""
enabled = "false"
enabled_modules = ""
mutex_trace = "false"
pin_codecache_trace = "false"
stack_trace = "false"

[loop_tracer]
iter_count = 36
iter_start = 0

[network]
collect_traffic_matrix = "false"
memory_model_1 = "bus"
memory_model_2 = "bus"
system_model = "magic"

[network/bus]
bandwidth = 25.6
ignore_local_traffic = "true"

[network/bus/queue_model]
type = "contention"

[network/emesh_hop_by_hop]
concentration = 1
dimensions = 2
hop_latency = 2
link_bandwidth = 64
size = ""
wrap_around = "false"

[network/emesh_hop_by_hop/broadcast_tree]
enabled = "false"

[network/emesh_hop_by_hop/queue_model]
enabled = "true"
type = "history_list"

[network/emesh_hop_counter]
hop_latency = 2
link_bandwidth = 64

[osemu]
clock_replace = "true"
nprocs = 0
pthread_replace = "false"
time_start = 1337000000

[perf_model]

[perf_model/branch_predictor]
mispredict_penalty = 8
size = 1024
type = "pentium_m"

[perf_model/cache]
levels = 3

[perf_model/core]
core_model = "nehalem"
frequency = 1.0
logical_cpus = 1
type = "interval"

[perf_model/core/interval_timer]
dispatch_width = 4
issue_contention = "true"
issue_memops_at_dispatch = "false"
lll_cutoff = 30
lll_dependency_granularity = 64
memory_dependency_granularity = 8
num_outstanding_loadstores = 10
window_size = 128

[perf_model/core/static_instruction_costs]
add = 1
branch = 1
delay = 0
div = 18
dynamic_misc = 1
fadd = 3
fdiv = 6
fmul = 5
fsub = 3
generic = 1
jmp = 1
mem_access = 0
mul = 3
recv = 1
spawn = 0
string = 1
sub = 1
sync = 0
tlb_miss = 0
unknown = 0

[perf_model/dram]
chips_per_dimm = 8
controller_positions = ""
controllers_interleaving = 4
dimms_per_controller = 4
direct_access = "false"
latency = 45
num_controllers = -1
per_controller_bandwidth = 7.6
type = "constant"

[perf_model/dram/cache]
enabled = "false"

[perf_model/dram/normal]
standard_deviation = 0

[perf_model/dram/queue_model]
enabled = "true"
type = "history_list"

[perf_model/dram_directory]
associativity = 16
directory_cache_access_time = 10
directory_type = "full_map"
home_lookup_param = 6
interleaving = 1
locations = "dram"
max_hw_sharers = 64
total_entries = 1048576

[perf_model/dram_directory/limitless]
software_trap_penalty = 200

[perf_model/dtlb]
associativity = 4
size = 64

[perf_model/fast_forward]
model = "oneipc"

[perf_model/fast_forward/oneipc]
include_branch_misprediction = "false"
include_memory_latency = "false"
interval = 100000

[perf_model/itlb]
associativity = 4
size = 128

[perf_model/l1_dcache]
address_hash = "mask"
associativity = 8
cache_block_size = 64
cache_size = 32
data_access_time = 4
dvfs_domain = "core"
next_level_read_bandwidth = 0
outstanding_misses = 0
passthrough = "false"
perf_model_type = "parallel"
perfect = "false"
prefetcher = "none"
replacement_policy = "lru"
shared_cores = 1
tags_access_time = 1
writeback_time = 0
writethrough = 0

[perf_model/l1_dcache/atd]

[perf_model/l1_icache]
address_hash = "mask"
associativity = 4
cache_block_size = 64
cache_size = 32
coherent = "true"
data_access_time = 4
dvfs_domain = "core"
next_level_read_bandwidth = 0
passthrough = "false"
perf_model_type = "parallel"
perfect = "false"
prefetcher = "none"
replacement_policy = "lru"
shared_cores = 1
tags_access_time = 1
writeback_time = 0
writethrough = 0

[perf_model/l1_icache/atd]

[perf_model/l2_cache]
address_hash = "mask"
associativity = 8
cache_block_size = 64
cache_size = 256
data_access_time = 8
dvfs_domain = "core"
next_level_read_bandwidth = 0
passthrough = "false"
perf_model_type = "parallel"
perfect = "false"
prefetcher = "none"
replacement_policy = "lru"
shared_cores = 1
tags_access_time = 3
writeback_time = 50
writethrough = 0

[perf_model/l2_cache/atd]

[perf_model/l3_cache]
address_hash = "mask"
associativity = 16
cache_block_size = 64
cache_size = 8192
data_access_time = 30
dvfs_domain = "global"
passthrough = "false"
perf_model_type = "parallel"
perfect = "false"
prefetcher = "none"
replacement_policy = "lru"
shared_cores = 4
tags_access_time = 10
writeback_time = 0
writethrough = 0

[perf_model/l3_cache/atd]

[perf_model/l4_cache]
passthrough = "false"
perfect = "false"

[perf_model/llc]
evict_buffers = 8

[perf_model/nuca]
enabled = "false"

[perf_model/stlb]
associativity = 4
size = 512

[perf_model/sync]
reschedule_cost = 1000

[perf_model/tlb]
penalty = 30
penalty_parallel = "true"

[power]
technology_node = 45
vdd = 1.2

[progress_trace]
enabled = "false"
filename = ""
interval = 5000

[queue_model]

[queue_model/basic]
moving_avg_enabled = "true"
moving_avg_type = "arithmetic_mean"
moving_avg_window_size = 1024

[queue_model/history_list]
analytical_model_enabled = "true"
max_list_size = 100

[queue_model/windowed_mg1]
window_size = 1000

[routine_tracer]
type = "none"

[sampling]
enabled = "false"

[scheduler]
type = "pinned"

[scheduler/big_small]
debug = "false"
quantum = 1000000

[scheduler/pinned]
core_mask = 1
interleaving = 1
quantum = 1000000

[scheduler/roaming]
core_mask = 1
quantum = 1000000

[scheduler/static]
core_mask = 1

[tags]

[traceinput]
address_randomization = "false"
enabled = "false"
mirror_output = "false"
num_runs = 1
restart_apps = "false"
stop_with_first_app = "true"
trace_prefix = ""

