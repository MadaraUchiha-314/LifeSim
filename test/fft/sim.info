{'benchmarks_revision': ("sh: 1: cd: can't cd to /home/pr16/sniper/sniper-6.1/benchmarks",),
 'benchmarks_root': ('/home/pr16/sniper/sniper-6.1/benchmarks',),
 'cmdline': ['../../run-sniper',
             '-n',
             '4',
             '-c',
             'gainestown',
             '-senergystats',
             '--roi',
             '--viz',
             '--',
             './fft',
             '-p',
             '4'],
 'git_revision': '',
 'host': 'PR16',
 'pin_version': 71313,
 'rusage': (32.932,
            3.112,
            905496,
            0,
            0,
            0,
            2437144,
            50,
            0,
            13200,
            68624,
            0,
            0,
            0,
            47257,
            308),
 'snipercmd': ['bash',
               '-c',
               '/home/pr16/sniper/sniper-6.1/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /home/pr16/sniper/sniper-6.1/lib/pin_sim -c /home/pr16/sniper/sniper-6.1/config/base.cfg --general/total_cores=4 --general/output_dir=/home/pr16/sniper/sniper-6.1/test/fft --config=/home/pr16/sniper/sniper-6.1/config/nehalem.cfg --config=/home/pr16/sniper/sniper-6.1/config/gainestown.cfg -g --general/magic=true -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/pr16/sniper/sniper-6.1/test/fft/sim.scripts.py -g --hooks/script0args= -- ./fft -p 4'],
 't_elapsed': 30.951987981796265,
 't_start': 1484189437.67515,
 'tracecmds': [],
 'user': 'pr16',
 'vmem': 1727053824L}
