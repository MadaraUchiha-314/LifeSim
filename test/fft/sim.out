                                     | Core 0     | Core 1     | Core 2     | Core 3    
  Instructions                       |     289646 |     288035 |     288074 |     288071
  Cycles                             |     292718 |     292718 |     292718 |     292718
  IPC                                |       0.99 |       0.98 |       0.98 |       0.98
  Time (ns)                          |     292718 |     292718 |     292718 |     292718
  Idle time (ns)                     |      70664 |      68448 |      69156 |      69554
  Idle time (%)                      |      24.1% |      23.4% |      23.6% |      23.8%
Branch predictor stats               |            |            |            |           
  num correct                        |       8418 |       8262 |       8263 |       8275
  num incorrect                      |        668 |        625 |        625 |        615
  misprediction rate                 |      7.35% |      7.03% |      7.03% |      6.92%
  mpki                               |       2.31 |       2.17 |       2.17 |       2.13
TLB Summary                          |            |            |            |           
  I-TLB                              |            |            |            |           
    num accesses                     |      27120 |      27073 |      27086 |      27083
    num misses                       |          2 |         27 |         27 |         27
    miss rate                        |      0.01% |      0.10% |      0.10% |      0.10%
    mpki                             |       0.01 |       0.09 |       0.09 |       0.09
  D-TLB                              |            |            |            |           
    num accesses                     |     169399 |     168690 |     168706 |     168701
    num misses                       |         14 |         34 |         34 |         37
    miss rate                        |      0.01% |      0.02% |      0.02% |      0.02%
    mpki                             |       0.05 |       0.12 |       0.12 |       0.13
  L2 TLB                             |            |            |            |           
    num accesses                     |         16 |         61 |         61 |         64
    num misses                       |         16 |         61 |         61 |         61
    miss rate                        |    100.00% |    100.00% |    100.00% |     95.31%
    mpki                             |       0.06 |       0.21 |       0.21 |       0.21
Cache Summary                        |            |            |            |           
  Cache L1-I                         |            |            |            |           
    num cache accesses               |      27120 |      27073 |      27086 |      27083
    num cache misses                 |        220 |        270 |        269 |        266
    miss rate                        |      0.81% |      1.00% |      0.99% |      0.98%
    mpki                             |       0.76 |       0.94 |       0.93 |       0.92
  Cache L1-D                         |            |            |            |           
    num cache accesses               |     169527 |     168771 |     168787 |     168783
    num cache misses                 |        587 |        603 |        598 |        593
    miss rate                        |      0.35% |      0.36% |      0.35% |      0.35%
    mpki                             |       2.03 |       2.09 |       2.08 |       2.06
  Cache L2                           |            |            |            |           
    num cache accesses               |        967 |       1003 |        998 |        990
    num cache misses                 |        408 |        750 |        749 |        741
    miss rate                        |     42.19% |     74.78% |     75.05% |     74.85%
    mpki                             |       1.41 |       2.60 |       2.60 |       2.57
  Cache L3                           |            |            |            |           
    num cache accesses               |        586 |        897 |        896 |        888
    num cache misses                 |        273 |        181 |        145 |        186
    miss rate                        |     46.59% |     20.18% |     16.18% |     20.95%
    mpki                             |       0.94 |       0.63 |       0.50 |       0.65
DRAM summary                         |            |            |            |           
  num dram accesses                  |        783 |          0 |          0 |          0
  average dram access latency (ns)   |      66.00 |        inf |        inf |        inf
  average dram queueing delay        |      12.57 |       0.00 |       0.00 |       0.00
  average dram bandwidth utilization |      2.00% |      0.00% |      0.00% |      0.00%
Coherency Traffic                    |            |            |            |           
  num loads from dram                |         68 |         75 |         72 |         74
  num loads from dram cache          |          0 |          0 |          0 |          0
  num loads from remote cache        |          0 |          0 |          0 |          0
